tests/phpunit/tests/Hooks/TestFrontend.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace WPML\PB\Elementor\Hooks;
+
+/**
+ * @group hooks
+ * @group frontend
+ * @group wpmlcore-7279
+ */
+class TestFrontend extends \OTGS_TestCase {
+
+	/**
+	 * @test
+	 */
+	public function TestAddHooks() {
+		$frontend = new Frontend();
+		\WP_Mock::expectActionAdded( 'elementor_pro/search_form/after_input', [ $frontend, 'elementor_add_language_form_field' ] );
+		$frontend->add_hooks();
+    }
+}
```

Comment:
@GeorgeBotsev, if you want to cover the whole code (and we want this :), you also need a test on the method `addLanguageFormField`:

```php
public function itShouldAddLanguageFormField() {
     $subject = new Frontend();
    
    \WP_Mock::expectAction( 'wpml_add_language_form_field' );

    $subject->add_hooks();
}
```